src/components/ChartRight/ChartRight.tsx

```diff
@@ -0,0 +1,27 @@
+import { memo } from 'react';
+import * as styled from './ChartRight.styles';
+import { useRecoilValue } from 'recoil';
+import { selectedCoinInfoState } from 'recoil/atoms/tableAtoms';
+import { ChartCoinInfo } from './ChartCoinInfo';
+import { ChartContent } from './ChartContent';
+
+function ChartRight() {
+  const selectedCoinInfo = useRecoilValue(selectedCoinInfoState);
+  return (
+    <styled.ChartContainer>
+      {selectedCoinInfo !== null && (
+        <ChartCoinInfo
+          symbol={selectedCoinInfo.symbol}
+          thumbnail={selectedCoinInfo.thumbnail}
+          coinName={selectedCoinInfo.coinName}
+          tradePrice={selectedCoinInfo.tradePrice}
+          changeRatio={selectedCoinInfo.changeRatio}
+          changePrice={selectedCoinInfo.changePrice}
+        />
```

Comment:
변수명이 잘 짜여져서 구조 분해를 사용하면 깔끔해질 것 같습니다.

```suggestion
        <ChartCoinInfo
          {...selectedCoinInfo}
        />
```